cli/command/service/formatter.go

```diff
@@ -122,6 +122,19 @@ Secrets:
  Target:	{{$secret.File.Name}}
   Source:	{{$secret.SecretName}}
 {{- end }}{{ end }}
+
```

Comment:
I wonder if we need/want those empty lines in the template? 🤔

Reply:
Ah, yes, it's a bit redundant as we have the indentation already (at least inconsistent). Let me clone this one and update that